config/default.yml

```diff
@@ -4493,6 +4493,11 @@ Style/StderrPuts:
   Enabled: true
   VersionAdded: '0.51'
 
+Style/StringChars:
+  Description: 'checks for uses of `String#split` with empty string or regexp literal argument.'
```

Comment:
I guess you should mention that the cops is not safe and add some notes about the safety.

Reply:
Actually, probably it's quite unlikely that some other class would define a split method that takes exactly the same args. :-)

Reply:
Thank you for the review. I marked the cop as unsafe and added a description to safety.
Initially, I didn't mark it as unsafe because I think it's less likely to be false positives 😅